transaction.js

```diff
@@ -0,0 +1,146 @@
+const { MongoClient } = require('mongodb');
+
+// CRUD operations in transactions must be on existing collections, so be sure you have run
+// usersCollection.js prior to running this script. 
+
+async function main() {
+    /**
+     * Connection URI. Update <username>, <password>, and <your-cluster-url> to reflect your cluster.
+     * See http://bit.ly/NodeDocs_lauren for more details
+     */
+    const uri = "mongodb+srv://<username>:<password>@<your-cluster-url>/test?retryWrites=true&w=majority";
```

Comment:
no need to specify `retryWrites=true`, it is true by default now

Reply:
When you get your connection string in Atlas, it provides it to you in this format, so I've included it that way here to avoid confusion.